dist/index.d.ts

```diff
@@ -2,3 +2,46 @@ import {Parser} from "lezer"
 
 export const parser: Parser
 export const promQLSyntax: { [key: string]: string[] }
+export const
+    LineComment: number,
+    Expr : number,
```

Comment:
The spacing around the colon is inconsistent, could you make them all `foo: bar` vs. `foo : bar`?

Reply:
done